organizational/conduct.rst

```diff
@@ -0,0 +1,71 @@
+Code of Conduct
+===============
+
+Hacker Society [#hacsoc]_ is, as Tim wrote in :doc:`hacsoc`, about an open
+society:
+
+  An open society promotes freedom of thought and expression, personal
+  responsibility, individualism, cooperation, and equality of all citizens.
+
+As such, we expect that our members behave in a way that allows us to achieve
+this.  Participants in HacSoc events are expected to treat each other with
+dignity and respect for each other's perspectives, experiences, and knowledge.
+In particular, participants should recognize that people of all skill levels are
+welcome, and should always be willing to help others learn.
+
+We want Hacker Society to be a place where people can be comfortable regardless
+of gender, gender identity and expression, sexual orientation, disability,
+physical appearance, body size, race, age, or religion.  As such, we do not
+tolerate harassment.  This includes, but is not limited to:
+
+- Rude or unwelcome comments
```

Comment:
@brenns10 That looks better. I also don't know exactly what they meant by "structures of domination." That seems to be pulling in a philosophical theory that I am unfamiliar with. I would go with:

> Rude or unwelcome comments about gender, gender identity, gender expression, sexual orientation, disability, physical appearance, attire, race, age or religious expression.

> Rude or unwelcome comments about a person's gender, gender identity, gender expression, sexual orientation, disability, physical appearance, attire, race, age or religious expression.

It might be valuable to have some examples and have some discussion of the tension in the document itself.